src/runner/browser-env/vite/browser-modules/expect.ts

```diff
@@ -0,0 +1,96 @@
+import { expect, type Expect } from "expect";
```

Comment:
Here I use `expect` from jest which is not installed as explicit dependency (installed from `expect-webdriverio`)